configure.ac

```diff
@@ -4,7 +4,7 @@ AC_PREREQ([2.5])
 ##########################################################################
 m4_define([odpapi_generation_version], [1])
 m4_define([odpapi_major_version], [23])
-m4_define([odpapi_minor_version], [5])
+m4_define([odpapi_minor_version], [6])
```

Comment:
Don't we want to wait for other pull requests (i.e. #965 and #972) before bumping the version? Version bump could be a separate commit.

Reply:
When those are ready, I'll push 972, 965 and then this patch set (back-to-back).